button-merchant/src/main/java/com/usebutton/merchant/ButtonInternal.java

```diff
@@ -58,6 +60,6 @@ void removeAttributionTokenListener(ButtonRepository buttonRepository, @NonNull
     void handlePostInstallIntent(ButtonRepository buttonRepository,
             PostInstallIntentListener listener, String packageName, DeviceManager deviceManager);
 
-    void reportOrder(ButtonRepository buttonRepository, DeviceManager deviceManager, Order order,
-            @Nullable OrderListener orderListener);
+    void reportOrder(ButtonRepository buttonRepository, DeviceManager deviceManager,
+            Features features, Order order, @Nullable OrderListener orderListener);
```

Comment:
I'm not particularly a fan of all the dependencies that we need to pass just to report an order. I understand this is only internal but still -- seems a bit bloated.